draft-ietf-quic-reliable-stream-reset.md

```diff
@@ -122,50 +123,62 @@ Reliable Size:  A variable-length integer indicating the amount of
 If the Reliable Size is larger than the Final Size, the receiver MUST close the
 connection with a connection error of type FRAME_ENCODING_ERROR.
 
-Semantically, a RESET_STREAM frame is equivalent to a RELIABLE_RESET_STREAM
-frame with the Reliable Size set to 0.
+CLOSE_STREAM frames are ack-eliciting. When lost, they MUST be retransmitted,
+unless another frame that causes a state transition to the Data Sent or Reset
+Sent State was sent for the same stream (see {{multiple-frames}}).
```

Comment:
How about talking about acknowledgement and terminal states?

```suggestion
unless the stream state has transitioned to "Data Recvd" or "Reset Recvd" due
to transmission and acknowledgement of other frames (see {{multiple-frames}}).
```

IMO, current text is a bit ambiguous in situations like when you send STREAM + FIN, then CLOSE_STREAM, after which you receive an ACK for STREAM + FIN. In such case, acknowledgment of STREAM + FIN changes the stream state to "Data Recvd" at which point the sender learns that retransmission of CLOSE_STREAM is unnecessary.